### PR TITLE
Address missing codebook info

### PR DESCRIPTION
**Pull request by evandragich**
This closes issue #25 ; I found the source data's original codebook and confirmed that 1 - urban, 2 - rural. This was opposite what the tutorial had, so I edited that to reflect.

### CODE REVIEW

File: inst/tutorials/06-dcbikeshare/06-dcbikeshare.Rmd [hunk 1]
```diff
@@ -312,7 +312,7 @@ dcbikeshare <- ___ %>%
 ```
 
 ```{r recode-season-exercise-hint-1}
-Use case_when() to mutate() the variable and fct_relevel() to convert it into a factor!
+Use 'case_when()' to 'mutate()' the variable and 'fct_relevel()' to convert it into a factor!
```
Comment (mine-cetinkaya-rundel) [inline inst/tutorials/06-dcbikeshare/06-dcbikeshare.Rmd]: These should be back ticks not single quotes

File: inst/tutorials/06-dcbikeshare/06-dcbikeshare.Rmd [hunk 2]
```diff
@@ -365,7 +365,7 @@ dcbikeshare <- ___ %>%
 ```
 
 ```{r recode-holiday-workingday-hint-1}
-First mutate() the variables using ifelse(), then use fct_relevel()
+First 'mutate()' the variables using 'ifelse()', then use 'fct_relevel()'
```
Comment (mine-cetinkaya-rundel) [inline inst/tutorials/06-dcbikeshare/06-dcbikeshare.Rmd]: Similarly here, backticks

File: inst/tutorials/07-exploregss/07-exploregss.Rmd [hunk 3]
```diff
@@ -287,7 +290,7 @@ Now, find the mean and median number of minutes respondents spend on email weekl
 ```
 
 ```{r mean-median-email-hint-1}
-Remember - you'll need to remove the NA values.
+Remember - you'll need to remove the 'NA' values.
```
Comment (mine-cetinkaya-rundel) [inline inst/tutorials/07-exploregss/07-exploregss.Rmd]: And backticks here too

**Review by mine-cetinkaya-rundel — CHANGES_REQUESTED** [review]
Just a few minor comments then this should be good to go!

**Comment by evandragich** [comment]
@mine-cetinkaya-rundel single quotes have been amended!

**Comment by evandragich** [comment]
whoops, my bad, `pr_finish()`ed on autopilot

**Comment by evandragich** [comment]
Finally done!

**Comment by evandragich** [comment]
> Finally done!

@mine-cetinkaya-rundel

**Pull request merged**